tips/migrations/0003_rename_message_tip.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.0.3 on 2022-03-30 18:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tips', '0002_test_data'),
+    ]
+
```

Comment:
Why do we change the model name?
I think this code is not needed for this app @orIsrael

Reply:
The initial name was "Text" when I changed it and make migrations this file was generated by Django.